portfolio/src/main/webapp/nav.js

```diff
@@ -0,0 +1,21 @@
+/* change src of iframe according to input index
+*/
```

Comment:
For functions, use a jsdoc comment. See https://google.github.io/styleguide/jsguide.html#jsdoc-general-form.

In general, make sure you're using a sentence-like structure when you write comments by using proper capitalization and punctuation. See https://google.github.io/styleguide/jsguide.html#jsdoc-general-form

```
/** Change the source of the iframe according to the index. */
```